File: Software Technologies/Exam Preparation SoftUni Babies Project/CSharp/SoftUniBabies/Models/Baby.cs
namespace SoftUniBabies.Models
{
    using System;
    using System.ComponentModel.DataAnnotations;

    public class Baby
    {
        //TODO

        [Key]
        public int Id { get; set; }

        public string Name { get; set; }

        public string Gender { get; set; }

        public string Birthday { get; set; }
    }
}

File: Software Technologies/Exam Preparation SoftUni Babies Project/JavaScript/models/baby.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    // TODO FIXME

    let Baby = sequelize.define('Baby', {
        name:{type: Sequelize.STRING, required: true, allowNull: false,},
        gender:{type: Sequelize.STRING, required: true, allowNull: false,},
        birthday:{type: Sequelize.STRING, required: true, allowNull: false,}
    }, {
        timestamp:false
    });

    return Baby;
};